Cargo.toml

```diff
@@ -126,32 +120,25 @@ iced-x86 = { version = "1.18.0", default-features = false, features = [
 ] }
 im = "15.1.0"
 im-rc = "15.1.0"
-indexmap = "2.1.0"
 indoc = "1.0.9"
 insta = "1.28.0"
 js-sys = "0.3.61"
 lazy_static = "1.4.0"
 libc = "0.2.139" # update roc_std/Cargo.toml on change
-libfuzzer-sys = "0.4"
```

Comment:
I would guess this is used but in a special workflow that is only run locally/manually when people fuzz.

Reply:
The only other place it's used is `crates/compiler/test_syntax/fuzz/Cargo.toml` but there it is specified as "0.4" not `workspace = true`. I can try to make it depend on the root `Cargo.toml` instead?

Reply:
Workspace sounds good to me, I suspect we'll be adding more fuzzers around the place.

Reply:
hmm, it looks like it's intentionally not part of the main workspace? https://github.com/roc-lang/roc/blob/main/crates/compiler/test_syntax/fuzz/Cargo.toml#L19-L21

Reply:
I added the dependency back, and added `test_syntax-fuzz` to the root workspace. It builds okay on my pc, we'll see if CI has any issues with building it :eyes: .